lib/bracket-matcher.coffee

```diff
@@ -81,7 +82,7 @@ class BracketMatcher
       @editor.moveRight()
       false
     else if autoCompleteOpeningBracket
-      @editor.insertText(text + pair)
+      @origEditorInsertText(text + pair)
       @editor.moveLeft()
```

Comment:
@jeancroy `adviceBefore` is how bracket matcher intercepts user input and reacts to opening brackets by automatically inserting the closing ones. Commenting that line out would disable bracket matcher.